### PR TITLE
Quick fix for signing failures

### PR DESCRIPTION
**Pull request by agocke**
Reverts the change to use the PortableStrongNameProvider and
switches us back to using the COM-based signing provider.

### CODE REVIEW

**Comment by agocke** [comment]
This is a temporary workaround for https://github.com/dotnet/roslyn/issues/23521

**Comment by agocke** [comment]
@MeiChin-Tsai for ask-mode approval.

**Comment by jaredpar** [comment]
@agocke test failures look real and just reacting to the default change. We should disable them, mark it with this bug and note that we need ot unskip the tests when fixing this bug.

**Comment by agocke** [comment]
ping @dotnet/roslyn-compiler for review -- this should be a very simple change

File: build/Targets/Imports.targets [hunk 1]
```diff
@@ -140,8 +140,9 @@
             <PublicKey>$(RoslynInternalKey)</PublicKey>
             <PublicKeyToken>fc793a00266884fb</PublicKeyToken>
           </PropertyGroup>
-          <!-- Real-signing is only supported cross-platform in the boostrap toolset -->
-          <PropertyGroup Condition="'$(OS)' != 'Windows_NT' AND '$(BootstrapBuildPath)' == ''">
+          <!-- Real-signing cross-platform currently has a blocking bug:
+               https://github.com/dotnet/roslyn/issues/23521 -->
```
Comment (jasonmalinowski) [inline build/Targets/Imports.targets]: The deep irony of disabling signing while simultaneously fixing the bug.... :smile:

**Comment by MeiChin-Tsai** [comment]
approved.

**Pull request merged**